.github/workflows/test.yaml

```diff
@@ -79,6 +79,25 @@ jobs:
         env:
           RUST_BACKTRACE: 1
 
+      - name: Install Rust nightly
+        if: matrix.os == 'ubuntu-latest'
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+          override: true
+
+      - name: Unit tests with address sanitizer
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          yarn build:test
+          LD_PRELOAD=/usr/lib/gcc/x86_64-linux-gnu/9/libasan.so yarn test
+        env:
+          RUST_TARGET: x86_64-unknown-linux-gnu
```

Comment:
Maybe cli should use the `CARGO_BUILD_TARGET` env var instead.

https://doc.rust-lang.org/cargo/reference/environment-variables.html#configuration-environment-variables